### PR TITLE
Release/2018.02 50 gef2fc6d4e

### PR DESCRIPTION
**Pull request by thepwagner**
Release notes + some consistency compulsions from previous release notes.

### CODE REVIEW

File: source/proxy-release-notes.md [hunk 1]
```diff
@@ -34,7 +43,7 @@ order: 20
 * Fixed several issues with timestamps included in reports sent to engine backend.
 * Added the ability to dump stacktraces of all running threads when engineproxy receives a `SIGUSR2` signal.
   When requested, traces are dumped to stderr. This should not be necessary unless requested by Apollo support.
-* Added the ability to collect performance data from engine proxy using [Go pprof profiler](https://golang.org/pkg/net/http/pprof/).
+* Added the ability to collect performance data from Engine proxy using [Go pprof profiler](https://golang.org/pkg/net/http/pprof/).
```
Comment (zionts) [inline source/proxy-release-notes.md]: There is a lot of inconsistency wherever we say "engine proxy" or "Engine Proxy" or "Engine proxy" or "engineproxy" – should we change all of them to match "Engine proxy" and note the change somewhere?

File: source/proxy-release-notes.md [hunk 2]
```diff
@@ -3,19 +3,28 @@ title: Proxy release notes
...
+* Added the ability to define the frontend port (the port Engine proxy will listen on) from an environment variable.
+  To define a frontend variable, remove `"port": 1234,` from the frontend configuration, and add `"portFromEnv": "MY_PORT_VARIABLE"`.
+  This will cause the proxy to read the `MY_PORT_VARIABLE` environment variable.
+  Heroku users in particular should set `"portFromEnv": "PORT"`.
+
 ### 2018.02-37-g678cbb68b – 2018-02-10
 
-* Adds support for GZIP content encoding for responses from Lambda origins.
-* Adds support for function qualifiers for Lambda origins.
+* Added support for GZIP content encoding for responses from Lambda origins.
```
Comment (zionts) [inline source/proxy-release-notes.md]: Thanks for fixing this consistency :)

File: source/proxy-release-notes.md [hunk 3]
```diff
@@ -3,19 +3,28 @@ title: Proxy release notes
 order: 20
 ---
 
+### 2018.02-50-gef2fc6d4e - 2018-02-15
+
+* Add support for proxying non-GraphQL requests with Lambda origins. This allows serving GraphiQL directly from a Lambda handler.
+  No additional configuration is required to start using this feature.
+* Added the ability to define the frontend port (the port Engine proxy will listen on) from an environment variable.
+  To define a frontend variable, remove `"port": 1234,` from the frontend configuration, and add `"portFromEnv": "MY_PORT_VARIABLE"`.
```
Comment (zionts) [inline source/proxy-release-notes.md]: "a frontend variable" seems weird to me – maybe "To define the frontend port via the environment, ..." ?

Comment (thepwagner) [reply source/proxy-release-notes.md]: I like that more - yoink.

File: source/proxy-release-notes.md (hunk 1 above)
Comment (thepwagner) [reply source/proxy-release-notes.md]: I don't want to take that on without clear "this is the brand" message - I definitely just picked `Engine proxy` randomly at one point.

I did fix a few `engineproxy`s in this file though, so it's at least internally consistent.

**Pull request merged**